contents/blog/how-to-measure-product-engagement.md

```diff
@@ -0,0 +1,101 @@
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: hanna-crombie
+featuredImage: ../images/blog/blog-generic-1.png
+featuredImageType: standard
+---
+
+Product engagement is the most important factor when it comes to driving retention and revenue for your business. Though it seems easy to view success through a financial lens, revenue alone is rarely a clear indicator of a sustainable product. 
+
+In this article we’ll take a look at why strong customer engagement metrics are vital to the success of your product, how to define engagement for your platform, and how to use analytics tools to measure and build on the results.
+
+## What is product engagement?
+The golden rule for any product manager: [listen to your users](https://posthog.com/handbook/product/user-feedback).
+
+Product engagement metrics measure how active your users are and how they spend time with your product. They give valuable insight into what is resonating, and what’s not quite hitting the mark. Knowing more about the specifics of customer engagement can inform your product roadmap going forward, so you build more features that matter and don’t dilute your offering with ones that don’t. 
+
+Churn is a metric we associate closely with customer satisfaction but alone it isn’t sophisticated enough to explain _why_ your customers do or do not stay. It’s also a lagging metric – it can take at least 30 days just to understand that a customer is not coming back, during which you’ve lost precious time.
+
+Your users are constantly providing feedback through their actions; which features they interact with, how they scroll, where their mouse hovers. By interpreting those actions, you’ll better understand what keeps them on your platform and be able to build more of what they need and love.
+
+Product engagement also makes a great [North Star Metric](https://posthog.com/blog/north-star-metrics), as it’s usually a reliable forecast for retention and can provide a strong foundation for product development.
+
+<BorderWrapper>
+    <Quote
+        imageSource="/images/customers/anca.png"
+        size="md"
+        name="Anca Filip"
+        title="Head of Product, Mention Me"
+        quote={`“PostHog has helped us improve our product and get a much better understanding of our users than we've ever been able to before.”`}
+    />
+</BorderWrapper>
+
+## What are product engagement metrics?
+The engagement metrics you need to track will depend on what your business looks like and what your product is.
+
+In order to use the right product engagement metric for your organization you need a strong understanding of your customer personas. What does your ideal user look like? How do they interact with your platform? What are the revenue drivers attached to them?
+By examining your personas and how their individual actions influence revenue and growth, you can select the most appropriate engagement metrics for your product.
+
+### Common engagement metrics
```

Comment:
I think we could be more useful here like in Andy's article - 'if you are an x company, we suggest y metric'. Phrases like 'for some products' and 'other digital services' aren't really useful.